README.md

```diff
@@ -1,6 +1,6 @@
-# ÜeberIdentity
+# UeberIdentity
```

Comment:
I'm not sure the `Ü` needs to be removed from the README as it's present in other documentation.  But this could be changed to `Überauth Identity` for consistency sake.

Just for the record the `Ue` is only necessary when you're not using `Ü`.  It's either `Üeberauth` or `Ueberauth`.